.dbosignore

```diff
@@ -0,0 +1,2 @@
+dist/
+package-lock.json
```

Comment:
It's so weird that if I include my package-lock.json the deploy would fail with 
```
migrated lockfile from package-lock.json 

error: lockfile had changes, but lockfile is frozen 
```